src/test/scala/de/zalando/beard/renderer/BugSpec.scala

```diff
@@ -1,39 +0,0 @@
-package de.zalando.beard.renderer
-
-import com.typesafe.scalalogging.LazyLogging
-import org.scalatest.{Matchers, FunSpec}
-
-import scala.io.Source
-
-/**
-  * @author dpersa
-  */
-class BugSpec extends FunSpec with Matchers with LazyLogging {
```

Comment:
The goal of this test is to make sure that the bug never happens again... I guess you should rename it or move it to a better place, to better reflect what it tests, but you shouldn't remove it...